src/background.ts

```diff
@@ -81,7 +83,9 @@ async function handleNewTab(tab: chrome.tabs.Tab) {
     });
 }
 
-function handleTabUpdate(tabId: any, changeInfo: any, tab: any) {
+async function handleTabUpdate(tabId: any, changeInfo: any, tab: any) {
+  const enable = await getStorage<boolean>("isOn");
```

Comment:
enable or isOn? Maybe we should use a same name

Reply:
done